ibexa/commerce/4.5/config/routes/ibexa_payment.yaml

```
@@ -0,0 +1,7 @@
+ibexa.checkout:
```

Too extensive copy&paste 😉 :
```suggestion
ibexa.payment:
```